src/pages/_app.tsx

```diff
@@ -1,12 +1,27 @@
 import type { AppProps } from "next/app";
+import { QueryClient, QueryClientProvider } from "react-query";
+import "../styles/globals.scss";
 
-import StateManagement from "../context/StateManagement";
+const queryClient = new QueryClient();
 
 function App({ Component, pageProps }: AppProps) {
   return (
-    <StateManagement>
+    <QueryClientProvider client={queryClient}>
+      <div className="logos">
```

Comment:
Do you think this could be abstracted to something like a `<Header />` component?

Reply:
Yeah I can move the logos to a header component